src/components/SolicitacaoDeKitLanche/Container.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import { meusDados } from "../../services/perfil.service";
+import { getWorkingDays as getDiasUteis } from "../../services/workingDays.service";
+import SolicitacaoDeKitLanche from ".";
+
+class Container extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      meusDados: null,
+      proximos_dois_dias_uteis: null,
+      proximos_cinco_dias_uteis: null
+    };
+  }
+
+  componentDidMount() {
+    meusDados().then(response => {
+      const meusDados = response;
+      this.setState({
+        meusDados
+      });
+    });
```

Comment:
meusDados().then(meusDados => {
      this.setState({
        meusDados
      });
    });

Reply:
feito